src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -31,7 +30,6 @@ class LicenceConditionService(
   private val additionalConditionUploadDetailRepository: AdditionalConditionUploadDetailRepository,
   private val conditionFormatter: ConditionFormatter,
   private val licencePolicyService: LicencePolicyService,
-  private val communityOffenderManagerRepository: CommunityOffenderManagerRepository,
```

Comment:
this is nice - reduction in dependencies and removing behaviour unrelated to conditions out of here 👍

Reply:
Thank you.